auth-web/src/composables/affiliations-factory.ts

```diff
@@ -43,6 +43,9 @@ export const useAffiliations = () => {
   }
 
   const isBusinessAffiliated = (businessIdentifier: string): boolean => {
+    if (businessIdentifier === undefined) {
+      return false
+    }
```

Comment:
I would do `if (!businessIdentifier)` to handle the null case, too. (I think we never have id=0.)

Walk me through this. This functions returns false, meaning the business is not affiliated, meaning...? How did the user get here and what happens now?

Reply:
I think it has to do with another change that was missing - the corpNum wasn't being brought up

Reply:
I think it has to do with another change that was missing - the corpNum wasn't being brought up